projects/store/package.json

```diff
@@ -0,0 +1,18 @@
+{
+  "name": "store",
+  "version": "16.0.0",
+  "peerDependencies": {
+    "@angular/common": "^16.1.0",
+    "@angular/core": "^16.1.0",
+    "@ngrx/entity": "^16.1.0",
+    "@ngrx/store": "^16.1.0",
+    "lodash": "^4.17.21"
```

Comment:
Lodash wordt maar 2 keer gebruikt in deze PR, en da's de get. Misschien nuttig om gewoon de switch te maken naar Ramda?

De Lodash.get komt eigenlijk overeen met de Ramda.path:
https://ramdajs.com/docs/#path

Wel argumenten omkeren normaal gezien.

Reply:
Ik ga deze momenteel nog laten voor wat het is, maar misschien moeten we hier gezamelijk afkloppen wat we willen met de rest van het team.